schema_salad/tests/test_avro_names.py

```diff
@@ -0,0 +1,9 @@
+from schema_salad.schema import load_schema
+
+from .util import get_data
+
+
+def test_avro_loading() -> None:
```

Comment:
function and module level docstrings, please and thank you :-)

`make diff_pydocstyle_report` to confirm